### PR TITLE
in-line validation

### PR DESCRIPTION
**Pull request by msokoloff1**
Update mal validation to be compatible with new in line mask types.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,5 +18,5 @@ test-staging: build
 test-prod: build
 	docker run -it -v ${PWD}:/usr/src -w /usr/src \
 		-e LABELBOX_TEST_ENVIRON="prod" \
-		-e LABELBOX_TEST_API_KEY_PROD=${LABELBOX_TEST_API_KEY_PROD} \
+		-e LABELBOX_TEST_API_KEY_PROD=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJja2NjOWZtbXc0aGNkMDczOHFpeWM2YW54Iiwib3JnYW5pemF0aW9uSWQiOiJja2N6NmJ1YnVkeWZpMDg1NW8xZHQxZzlzIiwiYXBpS2V5SWQiOiJja3g3dHFuM2JheHJnMHpibzR4b20yc2RpIiwic2VjcmV0IjoiODY4NDUzYmUxNDFiMDM0ZTkxMTY4MDVkZDYyYjQ1YTMiLCJpYXQiOjE2Mzk1OTAzMTQsImV4cCI6MjI3MDc0MjMxNH0.9xPEH2NNOLkz6DMZm76k8riXwMaI3nmv0ZPGXFJwPbw \
```
Comment (jtsodapop) [inline Makefile:21]: think we need to remove this

File: tests/integration/test_labeling_frontend.py [hunk 2]
```diff
@@ -12,6 +12,8 @@ def test_get_labeling_frontends(client):
         where=LabelingFrontend.iframe_url_path ==
         target_frontend.iframe_url_path)
     for frontend in filtered_frontends:
+        if frontend.name != 'Editor':
```
Comment (jtsodapop) [inline tests/integration/test_labeling_frontend.py:15]: do we know that frontends[0] will always be a non V4 frontend?

**Review by jtsodapop — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (msokoloff1) [reply Makefile:21]: oops!

Comment (msokoloff1) [reply Makefile:21]: I just made the key invalid

File: tests/integration/test_labeling_frontend.py (hunk 2 above)
Comment (msokoloff1) [reply tests/integration/test_labeling_frontend.py:15]: oops. this wasn't supposed to be pushed. Yeah this is broken.

**Review by jtsodapop — APPROVED** [review]

**Pull request merged**